#!/Users/riya/Documents/Generative_AI_Project/venv/bin/python3.10
import sys
from lib2to3.main import main

sys.exit(main("lib2to3.fixes"))
